spec/requests/apartments_spec.rb

```diff
@@ -2,6 +2,26 @@
 
 RSpec.describe "Apartments", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of apartments" do
+      Apartment.create(
+        street: '3880 San Ramon Dr', 
+        city: 'Oceanside', 
+        state: 'California',
+        manager: 'Bob Smith',
+        email: 'bobsmith@hotmail.com',
+        price: '$3000',
+        bedrooms: 2, 
+        bathrooms: 1,
+        pets: 'No',
+        image: 'https://images1.forrent.com/i2/tb-tHmpjytjE7Kxrc1HHnqm26BNSFIVF5rwMKFAGatA/117/image.jpg',
+        user_id: 1
+      )
+
+      get '/apartments'
+
+      apartment = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
+      #expect(apartment.length).to eq 1
```

Comment:
Are you testing the payload?

Reply:
To my understanding no, I won't be able to test the payload until I finish building out the create functionality in the next section.

Reply:
You are manually creating an apartment in your test. This is just checking to see if you can see all the apartments currently in the db.